README.md

```diff
@@ -74,7 +74,7 @@ render() {
             data={this.state.listViewData}
             renderItem={ (data, rowMap) => (
                 <View style={styles.rowFront}>
-                    <Text>I am {data.item} in a SwipeListView</Text>
+                    <Text>I am {data.text} in a SwipeListView</Text>
```

Comment:
@JuniorBoaventura nice catch, I actually think we're both wrong here. I think it should actually be `data.item.text`, would you mind making that change?